.github/workflows/hackage.yml

```diff
@@ -109,6 +107,7 @@ jobs:
         run: |
           cd $(ls -d ./incoming/${{ matrix.package }}-*)
           # For brittany
+          echo "allow-newer:" >> cabal.project
```

Comment:
big hammer?

Reply:
nope, below there are the concrete allow newer

Reply:
nope, just below there are the concrete allow newers needed to make the solver happy, the section will look:

```
allow-newer:
 butcher:base, multistate:base, data-tree-print:base,
 stylish-haskell:Cabal,stylish-haskell:ghc-lib-parser,stylish-haskell:aeson
```

Reply:
So brittany cant be built from hackage and ghc-9.0.2, will update the hls.cabal file if needed

Reply:
No, it is a new cabal.project created on the fly. I think i broke it in a previous pr and i detected the error here